app/models/user.rb

```diff
@@ -54,25 +54,62 @@ def administrated_projects
     scope
   end
 
-  def self.to_csv
-    @users = User.order(:id)
+  def self.to_csv(
```

Comment:
this whole section might make a nice `UserCsvPresenter` class or something like that ... but can refactor that after it's merged ...